packages/back-end/types/visual-changeset.d.ts

```diff
@@ -0,0 +1,23 @@
+interface DOMMutation {
+  selector: string;
+  action: "append" | "set" | "remove";
+  attribute: string;
+  value: string;
+}
+
+interface VisualChange {
+  id: string;
+  description: string;
+  css: string;
+  variation: string;
```

Comment:
Assuming this is related to the recently-added variation ID, not sure if this will conflict with something @mknowlton89 did recently, but I think somewhere in the UI the variation ID is used only for reordering and is subject to change, so if variations are reordered, is it possible this will no longer match up with a DOM element? The suggested change wouldn't be anything here but rather to ensure that the reordering logic doesn't break any identifiers.

Reply:
I have seen us send erroneous properties from the front-end (like in this case, if `id` is only used for front-end drag-and-drop, it probably shouldn't be sent) and I have seen us take the entire payload and throw it into Mongo, so there's definitely a risk there. Whether we're doing that for the flows and endpoints that would impact variation ID's, I'm not sure.